frontend/components/Status.tsx

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ITask, Ilist } from "../src/App";
+
+function Status(props: {
+  submited: boolean;
+  setSubmited: (Submit: boolean) => void;
+  list: ITask[];
+  setList: (list: ITask[]) => void;
+}) {
+  const [complete, setComplete] = useState<ITask[]>([]);
+  const [pending, setPending] = useState<ITask[]>([]);
+  function splitList() {
+    for (let x in props.list) {
+      if (props.list[x].completed) {
+        setComplete((complete) => [...complete, props.list[x]]);
+      } else {
+        setPending((pending) => [...pending, props.list[x]]);
+      }
+    }
+  }
+  async function handleChange(id: number) {
+    let updateTask = await axios.put<Ilist>("updateTask/", { id: id });
```

Comment:
Here we create a variable: `let updateTask` but do not use it.

I'd expect you to check the response from the backend, and if the status code is 200, then we can let the user know.